main.py

```diff
@@ -13,11 +16,14 @@
 
 def Receive():
     print("start Reveive ...")
-    cap = cv2.VideoCapture(os.getenv("rtsp"))
-    cap.set(cv2.CAP_PROP_BUFFERSIZE, 2)
-    while cap.isOpened():
-        _, frame = cap.read()
-        deq.append(frame)
+    while True:
+        cap = cv2.VideoCapture(os.getenv("rtsp"))
+        cap.set(cv2.CAP_PROP_BUFFERSIZE, 2)
+        while cap.isOpened():
+            _, frame = cap.read()
+            deq.append(frame)
+        print("[{} UTC] deq is empty, retry...".format(datetime.datetime.utcnow()))
```

Comment:
print("[{} UTC] rtsp stream is closed, retry...".format(datetime.datetime.utcnow()))